src/components/staking/PageValidator.vue

```diff
@@ -245,6 +247,11 @@ export default {
     selfBond() {
       return percent(this.delegates.selfBond[this.validator.operator_address])
```

Comment:
should the formatting happen via filters in the DOM?

Reply:
don't know what you mean

Reply:
https://github.com/luniehq/lunie/blob/develop/src/ActionModal/components/ModalWithdrawRewards.vue#L21 @colw introduced the concept of filters. You pipe the data through filters to format them.

Reply:
oh yeah, that seems great. i didn't make this computed property this way -`selfBond` was already like this...